runtime/compiler/control/J9Options.cpp

```diff
@@ -128,6 +128,8 @@ int64_t J9::Options::_oldAgeUnderLowMemory = 1000*60*5; // 5 minute
 int64_t J9::Options::_timeBetweenPurges = 1000*60*1; // 1 minute
 bool J9::Options::_shareROMClasses = false;
 int32_t J9::Options::_sharedROMClassCacheNumPartitions = 16;
+int64_t J9::Options::_highActiveThreadThreshold = 63 * 0.8;
```

Comment:
That 63 should come from the maximum number of threads instead of being hardcoded.

Reply:
TR_Options::getNumUsableCompilationThreads()

Reply:
The code also needs to verify that those two new thresholds are never larger than the number of active compilation threads.